examples/operator/templates/deployment.yaml

```diff
@@ -98,6 +85,26 @@ spec:
           subPath: controller_manager_config.yaml
         - mountPath: /my.ca
           name: secret-volume
+        {{- if .Values.controllerManager.manager.livenessProbe }}
```

Comment:
should `Values.controllerManager.manager.livenessProbe` be in values.yaml if probe is presented in source manifest?

Reply:
please check that it is in values.yaml if flag is enabled. values.yaml was not updated, only template